.pylintrc

```diff
@@ -14,38 +14,7 @@ notes=FIXME,
 # Black handles line-too-long for us
```

Comment:
If things are added correctly to the `pyproject.toml`, it should be possible to remove the `.pylintrc` file entirely. You can refer to https://github.com/nautobot/nautobot-plugin-netbox-importer/blob/develop/pyproject.toml#L59 as an example.

Reply:
Thanks for this.I have removed the .pylintrc file now.